Packs/MISPFeed/Integrations/MISPFeed/Pipfile.lock

```diff
@@ -0,0 +1,369 @@
+{
```

Comment:
There is no specific package here, you can delete the contents of the file or the file entirely.

Reply:
fixed